platform.go

```diff
@@ -19,11 +19,16 @@ type platformPlugin struct {
 	glfwTasker *tasker.Tasker
 	window     *glfw.Window
 	channel    *plugin.MethodChannel
+
+	// flutterInitialized gets called once flutter is running and ready
+	// to process upstream plugin calls. (It usually takes ~10 rendering frame).
+	flutterInitialized func()
```

Comment:
I think this should either be named to be specifically used for initialRoute, so that it's clear other parts of the code (future PR's) should not overwrite it. Or it should be a slice of func()'s to which more than one functions can be appended.